### ISSUE TITLE
Create standalone Windows executable for schedule_viewer

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#3]
```
schedule_viewer is really cool, so we should make it easier for people to
run by creating a Windows executable version.


```

Original issue reported on code.google.com by `joe.hugh...@gmail.com` on 22 Apr 2007 at 1:09

**Comment by GoogleCodeExporter** [comment]
Original comment by `joe.hugh...@gmail.com` on 22 Apr 2007 at 1:13
- Added labels: **Language-Python**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with upload of transitfeed-1.0b6-windows.zip
```

Original comment by `tom.brow...@gmail.com` on 27 Apr 2007 at 3:19
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with upload of transitfeed-1.0b6-windows.zip
```

Original comment by `tom.brow...@gmail.com` on 27 Apr 2007 at 3:38
